loops.js

```diff
@@ -17,7 +19,14 @@
 // loop...
 // klaar!
  // ==========================================
-
+const loop = 'loop...';
+for (let i = 0; i < 5; i++) {
+    if (i < 4){
```

Comment:
Je kan de code korter maken door het zo te schrijven.

for (let i = 0; i < 4; ++i) {
    console.log("loop...");
    if (i === 3) {
        console.log("klaar!");
    }
}